packages/npm/src/definitions/index.ts

```diff
@@ -0,0 +1,5 @@
+// @ts-nocheck - generated by Theo
+export * from './borderRadius';
+export * from './colorPalette';
+export * from './themeColors';
+export type Color = ColorPalette | ThemeColors;
```

Comment:
This would fail as the types are not imported.

The `@ts-nocheck` comments I find dangerous as well.

Created a "refactor suggestions" PR to put that all into an understandable diff format

https://github.com/onfido/castor-tokens/pull/25